src/libraries/Microsoft.Extensions.Configuration.Binder/tests/ConfigurationCollectionBindingTests.cs

```diff
@@ -282,33 +282,6 @@ private void GetIntDictionaryT<T>(T k1, T k2, T k3)
             Assert.Equal("val_2", options[k3]);
         }
 
-        [Fact]
```

Comment:
These two tests are 100% duplicates with the `BindXXX` tests directly below them. No reason to have duplicate tests.

Reply:
Thanks for fixing so quickly